### PR TITLE
Ronniegeraghty inventory updates 6

### PR DESCRIPTION
**Pull request by ronniegeraghty**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -594,17 +594,17 @@
...
-"WindowsAzure.MobileServices.Backend.Storage","1.0.478","","WindowsAzure Mobile Services - Backend Storage","App Service","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
-"WindowsAzure.MobileServices.Backend.Tables","1.0.478","","WindowsAzure Mobile Services - Backend Tables","App Service","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
-"WindowsAzure.MobileServices.SQLiteStore","1.0.4","","WindowsAzure Mobile Services - SQLiteStore","App Service","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
-"WindowsAzure.MobileServices.WinJS","","2.0.0-beta","WindowsAzure Mobile Services - WinJS","App Service","NA","NA","NA","client","False","","","","","","","","","","",""
-"WindowsAzure.Storage","9.3.3","","WindowsAzure Storage","Storage","https://github.com/Azure/azure-storage-net","NA","NA","client","False","","","","deprecated","3/31/2023","","Azure.Storage.Common","NA","","",""
+"WindowsAzure.MobileServices","1.3.2","","WindowsAzure Mobile Services","Mobile Services","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:597]: @ronniegeraghty Looks like this is a bad merge as you didn't copy over the NA values that were recently added. That is what is causing the CI to fail.

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv:597]: @weshaggard what field needs to be set to NA? Looks like they all have NA for Repo Path.

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -594,17 +594,17 @@
...
+"WindowsAzure.MobileServices.Backend.SignalR","1.0.478","","WindowsAzure Mobile Services - Backend SignalR","Mobile Services","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
+"WindowsAzure.MobileServices.Backend.Storage","1.0.478","","WindowsAzure Mobile Services - Backend Storage","Mobile Services","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
+"WindowsAzure.MobileServices.Backend.Tables","1.0.478","","WindowsAzure Mobile Services - Backend Tables","Mobile Services","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
+"WindowsAzure.MobileServices.SQLiteStore","1.0.4","","WindowsAzure Mobile Services - SQLiteStore","Mobile Services","NA","NA","NA","client","False","","","","deprecated","3/31/2023","","NA","NA","","",""
+"WindowsAzure.MobileServices.WinJS","","2.0.0-beta","WindowsAzure Mobile Services - WinJS","Mobile Services","NA","NA","NA","client","False","","","","","","","NA","NA","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:606]:
```suggestion
"WindowsAzure.MobileServices.WinJS","","2.0.0-beta","WindowsAzure Mobile Services - WinJS","Mobile Services","NA","NA","NA","client","False","","","","","","","","","","",""
```

Those weren't there before. Perhaps this should be deprecated but it currently isn't. @mario-guerra should we deprecate this package as well.

File: _data/releases/latest/dotnet-packages.csv [hunk 3]
```diff
@@ -594,17 +594,17 @@
...
+"WindowsAzure.Storage","9.3.3","","WindowsAzure Storage","Storage","https://github.com/Azure/azure-storage-net","NA","NA","client","False","","","","deprecated","3/31/2023","","Azure.Storage.Common","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:607]:
```suggestion
"WindowsAzure.Storage","9.3.3","","WindowsAzure Storage","Storage","https://github.com/Azure/azure-storage-net","NA","NA","client","False","","","","deprecated","3/31/2023","","Azure.Storage.Common","NA","","",""
```

**Pull request merged**